apollo-router/src/plugin/test/mod.rs

```diff
@@ -257,15 +198,16 @@ mod testing {
             Ok(Self {})
         }
     }
+    register_plugin!("apollo.test", "empty_plugin", EmptyPlugin);
 
     #[tokio::test]
     async fn test_test_harness() -> Result<(), BoxError> {
         let mut harness = PluginTestHarness::builder()
-            .plugin(EmptyPlugin {})
-            .schema(IntoSchema::Canned)
+            .configuration(json!({ "test.empty_plugin": null }))
             .build()
             .await?;
         let graphql = harness.call_canned().await?.next_response().await.unwrap();
+        assert_eq!(graphql.errors, []);
```

Comment:
nit
```suggestion
        assert!(graphql.errors.is_empty());
```

This imo would look more symetrical with the kinds of assertions [apollo-rs does](https://github.com/apollographql/apollo-rs/blob/233ed800417c00575c601722f5204bf91cace4be/crates/apollo-compiler/src/lib.rs#L211)